user-guide/Advanced_Modules/DOM/Tutorials/DOM_Remove_Enum_Entry.md

```diff
@@ -0,0 +1,406 @@
+using Skyline.DataMiner.Net.Apps.DataMinerObjectModel;
+using Skyline.DataMiner.Net.Apps.Modules;
+using Skyline.DataMiner.Net.Apps.Sections.SectionDefinitions;
+using Skyline.DataMiner.Net.GenericEnums;
+using Skyline.DataMiner.Net.ManagerStore;
+using Skyline.DataMiner.Net.Messages.SLDataGateway;
+using Skyline.DataMiner.Net.Sections;
+
+
+namespace Tutorial
+{
+    ///<summary>
+    /// This script will create a basic people app with one DomInstance.
+    /// Each domInstance for this sectionDefinition will contain:
+    ///     - first name of the person
+    ///     - last name of the person
+    ///     - phone number of the person
+    ///     - job title of the person
+    ///</summary>
+    public class Script
+    {
+        private const string ModuleName = "people_app";
+        private DomHelper _domHelper;
+        private Engine _engine;
+
+        public void Run(Engine engine)
+        {
+            _engine = engine;
+            _domHelper = new DomHelper(_engine.SendSLNetMessages, ModuleName);
+            
+            // Create the ModuleSettingsHelper
+            var moduleSettingsHelper = new ModuleSettingsHelper(_engine.SendSLNetMessages);
+
+            // Create the default moduleSettings and save it
+            var moduleSettings = new ModuleSettings(ModuleName);
+            moduleSettingsHelper.ModuleSettings.Create(moduleSettings);
+
+            // Start creating the DOM objects now that you have the domHelper
+            var firstNameFieldDescriptor = new FieldDescriptor()
+            {
+                FieldType = typeof(string),
+                IsOptional = false,
+                Name = "First name",
+                Tooltip = "First name of the person"
+            };
+
+            var lastNameFieldDescriptor = new FieldDescriptor()
+            {
+                FieldType = typeof(string),
+                IsOptional = false,
+                Name = "Last name",
+                Tooltip = "Last name of the person"
+            };
+
+            var phoneNumberFieldDescriptor = new FieldDescriptor()
+            {
+                FieldType = typeof(string),
+                IsOptional = false,
+                Name = "Phone number",
+                Tooltip = "Phone number of the person"
+            };
+
+            var genericEnum = new GenericEnum<int>();
+            genericEnum.AddEntry("Director", 0);
+            genericEnum.AddEntry("Project manager", 1);
+            genericEnum.AddEntry("Engineer", 2);
+            genericEnum.AddEntry("Secretary", 3);
+
+            var jobTitleFieldDescriptor = new GenericEnumFieldDescriptor()
+            {
+                FieldType = typeof(GenericEnum<int>),
+                IsOptional = true,
+                GenericEnumInstance = genericEnum,
+                Name = "JobTitle",
+                Tooltip = "Job title of the person"
+            };
+
+            // Put all the fieldDescriptors in a sectionDefinition
```

Comment:
Camel casing doesn't feel right imo, I would suggest going over the whole file and changing the naming from 'fieldDescriptor' to 'FieldDescriptor'.